Trigger `apiKeyDailyTask` throwing Unsupported field: DayOfMonth

**Describe the bug**

Seeing from the logs:
```
2025-03-01 11:45:53,011 dd620d5e2ac5 /usr/lib/jvm/java-17-openjdk-17.0.14.0.7-2.el9.x86_64/bin/java[8] ERROR [io.qua.sch.com.run.StatusEmitterInvoker] (executor-thread-1206) Error occurred while executing task for trigger IntervalTrigger [id=1_io.hyperfoil.tools.horreum.svc.UserServiceImpl#apiKeyDailyTask, interval=86400000]: java.util.concurrent.CompletionException: java.time.temporal.UnsupportedTemporalTypeException: Unsupported field: DayOfMonth
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:332)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:347)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:874)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:887)
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2325)
	at java.base/java.util.concurrent.CompletableFuture$MinimalStage.whenComplete(CompletableFuture.java:2902)
	at io.quarkus.scheduler.common.runtime.DefaultInvoker.invoke(DefaultInvoker.java:25)
	at io.quarkus.scheduler.common.runtime.DelegateInvoker.invokeDelegate(DelegateInvoker.java:28)
	at io.quarkus.scheduler.common.runtime.StatusEmitterInvoker.invoke(StatusEmitterInvoker.java:35)
	at io.quarkus.scheduler.runtime.SimpleScheduler$ScheduledTask.doInvoke(SimpleScheduler.java:472)
	at io.quarkus.scheduler.runtime.SimpleScheduler$ScheduledTask$2.call(SimpleScheduler.java:454)
	at io.quarkus.scheduler.runtime.SimpleScheduler$ScheduledTask$2.call(SimpleScheduler.java:451)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:270)
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:640)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.time.temporal.UnsupportedTemporalTypeException: Unsupported field: DayOfMonth
	at java.base/java.time.Instant.getLong(Instant.java:604)
	at java.base/java.time.format.DateTimePrintContext.getValue(DateTimePrintContext.java:308)
	at java.base/java.time.format.DateTimeFormatterBuilder$NumberPrinterParser.format(DateTimeFormatterBuilder.java:2763)
	at java.base/java.time.format.DateTimeFormatterBuilder$CompositePrinterParser.format(DateTimeFormatterBuilder.java:2402)
	at java.base/java.time.format.DateTimeFormatter.formatTo(DateTimeFormatter.java:1849)
	at java.base/java.time.format.DateTimeFormatter.format(DateTimeFormatter.java:1823)
	at io.quarkus.qute.runtime.extensions.TimeTemplateExtensions.format(TimeTemplateExtensions.java:32)
	at io.quarkus.qute.runtime.extensions.TimeTemplateExtensions_Extension_ValueResolver_format_20fff29d757fefef0946b2d3432680c31e8cb1e1$$function$$1.accept(Unknown Source)
	at io.quarkus.qute.CompletedStage.whenComplete(CompletedStage.java:285)
	at io.quarkus.qute.runtime.extensions.TimeTemplateExtensions_Extension_ValueResolver_format_20fff29d757fefef0946b2d3432680c31e8cb1e1.resolve(Unknown Source)
	at io.quarkus.qute.EvaluatorImpl.resolve(EvaluatorImpl.java:211)
	at io.quarkus.qute.EvaluatorImpl.resolveReference(EvaluatorImpl.java:131)
	at io.quarkus.qute.EvaluatorImpl.lambda$resolveReference$2(EvaluatorImpl.java:135)
	at io.quarkus.qute.CompletedStage.thenCompose(CompletedStage.java:249)
	at io.quarkus.qute.EvaluatorImpl.resolveReference(EvaluatorImpl.java:135)
	at io.quarkus.qute.EvaluatorImpl.evaluate(EvaluatorImpl.java:85)
	at io.quarkus.qute.ResolutionContextImpl.evaluate(ResolutionContextImpl.java:29)
	at io.quarkus.qute.ExpressionNode.resolve(ExpressionNode.java:36)
	at io.quarkus.qute.SectionNode$SectionResolutionContextImpl.execute(SectionNode.java:245)
	at io.quarkus.qute.SectionHelper$SectionResolutionContext.execute(SectionHelper.java:66)
	at io.quarkus.qute.Parser$1.resolve(Parser.java:1331)
	at io.quarkus.qute.SectionNode.resolve(SectionNode.java:53)
	at io.quarkus.qute.SectionNode.resolve(SectionNode.java:58)
	at io.quarkus.qute.TemplateImpl$TemplateInstanceImpl.renderData(TemplateImpl.java:244)
	at io.quarkus.qute.TemplateImpl$TemplateInstanceImpl.renderAsyncNoTimeout(TemplateImpl.java:235)
	at io.quarkus.qute.TemplateImpl$TemplateInstanceImpl.render(TemplateImpl.java:160)
	at io.quarkus.qute.runtime.TemplateProducer$InjectableTemplate$InjectableTemplateInstanceImpl.render(TemplateProducer.java:348)
	at io.hyperfoil.tools.horreum.notification.EmailPlugin$EmailNotification.notifyApiKeyExpiration(EmailPlugin.java:171)
	at io.hyperfoil.tools.horreum.svc.NotificationServiceImpl.lambda$notifyApiKeyExpiration$6(NotificationServiceImpl.java:204)
```

**Version**

What is the version of Horreum ?

If you are using a development branch; what is the commit id ?

```
0.17.1
```

--- a/horreum-backend/src/main/java/io/hyperfoil/tools/horreum/notification/EmailPlugin.java
+++ b/horreum-backend/src/main/java/io/hyperfoil/tools/horreum/notification/EmailPlugin.java
@@ -7,7 +7,7 @@ import java.text.DateFormat;
 import java.text.SimpleDateFormat;
 import java.time.Duration;
 import java.time.Instant;
-import java.time.temporal.ChronoUnit;
+import java.time.ZoneOffset;
 import java.util.Date;
 
 import jakarta.enterprise.context.ApplicationScoped;
@@ -164,8 +164,8 @@ public class EmailPlugin implements NotificationPlugin {
                     .data("baseUrl", baseUrl)
                     .data("username", username)
                     .data("keyName", keyName)
-                    .data("creation", creation.truncatedTo(ChronoUnit.DAYS))
-                    .data("lastAccess", lastAccess.truncatedTo(ChronoUnit.DAYS))
+                    .data("creation", creation.atOffset(ZoneOffset.UTC).toLocalDateTime())
+                    .data("lastAccess", lastAccess.atOffset(ZoneOffset.UTC).toLocalDateTime())
                     .data("expiration", toExpiration)
                     .data("active", active)
                     .render();

--- a/horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/NotificationPluginTest.java
+++ b/horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/NotificationPluginTest.java
@@ -63,6 +63,11 @@ public class NotificationPluginTest {
     }
 
     @Test
+    public void testApiKeyExpiration() {
+        withAllPlugins(notification -> notification.notifyApiKeyExpiration("Dummy key", Instant.now(), Instant.now(), 1, 1));
+    }
+
+    @Test
     public void test() {
         var event = new MissingValuesEvent(new DatasetDAO.Info(1, 1, 0, 1), new HashSet<>(Arrays.asList("foo", "bar")), true);
         withAllPlugins(notification -> notification.notifyMissingValues("Dummy Test", null, event));

